k8s/tools/admin-node/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:18.04
+
+RUN apt-get update\
+        && apt-get install -y --no-install-recommends mysql-client python-setuptools python-pip\
```

Comment:
Given the comment above, this would simplify to:
```sh
&& apt-get install -y --no-install-recommends mysql-client \
```

Reply:
> I was thinking that it may be useful to install openssl
I believe python:3 is an ubuntu image and it should come with openssl but it doesn't hurt to specify it on the dockerfile